build.sbt

```diff
@@ -61,23 +76,11 @@ lazy val publishSettings = Seq(
     username <- sys.env.get("SONATYPE_USERNAME")
     password <- sys.env.get("SONATYPE_PASSWORD")
   } yield Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)).toSeq,
-  publishTo := sonatypePublishToBundle.value,
+  publishTo := Some(
+    if (isSnapshot.value) Opts.resolver.sonatypeSnapshots else Opts.resolver.sonatypeStaging
+  ),
   releaseCrossBuild := true,
   releasePublishArtifactsAction := PgpKeys.publishSigned.value,
-  releaseProcess := Seq[ReleaseStep](
```

Comment:
Are you still seeing issues with this? I would like this to return; I tried it out and it seems to work as expected.

Reply:
At least `release skip-tests` doesn't work with this, and there's no way of just publishing to staging without actually releasing the artifacts for real, so there's no way to dry-run this? Not going to waste more time on this anymore.

Reply:
```
  releaseProcess := Seq[ReleaseStep](
    checkSnapshotDependencies,
    inquireVersions,
    runClean,
    ReleaseStep { st: State =>
      if (!st.get(ReleaseKeys.skipTests).getOrElse(false)) {
        releaseStepCommandAndRemaining("+test")(st)
      } else {
        st
      }
    },
    setReleaseVersion,
    commitReleaseVersion,
    tagRelease,
    releaseStepCommandAndRemaining("+publishSigned"),
    releaseStepCommand("sonatypeBundleRelease"),
    setNextVersion,
    commitNextVersion,
    pushChanges
  ),
```